### PR TITLE
date to isoDate

### PR DESCRIPTION
**Pull request by nicolaifsf**
The way we currently format our timestamps is the date-time format, however according to our own swagger spec, we state that this is simply the date format. this PR suggests a fix that would bring us to accordance with the swagger spec with the way we send timestamps.

### CODE REVIEW

**Comment by d2lam** [comment]
👍

File: models/pipeline.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ const MODEL = {
         .example('git@github.com:screwdriver-cd/optional-config.git#master'),
 
     createTime: Joi
-        .date()
+				.string()
```
Comment (jer) [inline models/pipeline.js]: Something weird happened to the formatting here

Comment (nicolaifsf) [inline models/pipeline.js]: hum. this is not the first time this has happened to me. 
I'll fix it.

**Comment by jer** [comment]
So is this a bug in hapi-swagger that is preventing it from defining the right json from .date()? It seems nicer to keep it a date object instead of holding on to a string if we can get away with it, but if this is the only way to make the swagger work, I'm 👍 until the underlying issue is resolved.

Also please squash the commits and force-push back if you could

**Comment by nicolaifsf** [comment]
That's a really good question. Specifically wether or not it actually is a bug or not.
So, hapi-swagger in their examples actually uses this string().isoDate() method in order to follow the ISO8601 standard for date-time formatting (how we currently present the dates) https://github.com/hapijs/joi/blob/master/test/string.js#L2567-L2609
But after doing some digging right now to answer your question, I found that in joi .date().iso() appears to have the same effect in specifying the standard that we are using.

**Pull request closed without merging**